draft-ietf-taps-interface.md

```diff
@@ -344,46 +384,53 @@ The Resolve() action on Preconnection can be used by the application to force
 
-## Specifying Transport Properties {#connection-props}
+## Specifying Transport Properties {#selection-props}
 
 A Preconnection Object holds properties reflecting the application's
 requirements and preferences for the transport. These include Selection
-Properties (Protocol and Path Selection Properties), as well as Generic and
-Specific Protocol Properties for configuration of the detailed operation of the
-selected Protocol Stacks.
-
-The protocol(s) and path(s) selected as candidates during Connection
-establishment are determined by a set of properties. Since there could be
-paths over which some transport protocols are unable to operate, or remote
-endpoints that support only specific network addresses or transports,
-transport protocol selection is necessarily tied to path selection. This may
-involve choosing between multiple local interfaces that are connected to
-different access networks.
+Properties for selecting protocol stacks and paths, as well as Connection
+Properties for configuration of the detailed operation of the selected Protocol
+Stacks.
+
+The protocol(s) and path(s) selected as candidates during establishment are
+determined and configufed using these properties. Since there could be paths
+over which some transport protocols are unable to operate, or remote endpoints
+that support only specific network addresses or transports, transport protocol
+selection is necessarily tied to path selection. This may involve choosing
+between multiple local interfaces that are connected to different access
+networks.
+
+Selection properties are represented as preferences, which can have one of five
+preference levels:
+
+   | Preference | Effect                                                                 |
+   |------------|------------------------------------------------------------------------|
+   | Require    | Select only protocols/paths providing the property, fail otherwise     |
+   | Prefer     | Prefer protocols/paths providing the property, proceed otherwise       |
+   | Ignore     | Cancel any system default preference for this property                 |
+   | Avoid      | Prefer protocols/paths not providing the property, proceed otherwise   |
+   | Prohibit   | Select only protocols/paths not providing the property, fail otherwise |
 
 Internally, the transport system will first exclude all protocols and paths
 that match a Prohibit, then exclude all protocols and paths that do not match
 a Require, then sort candidates according to Preferred properties, and then
-use Avoided properties as a tiebreaker. In case of conflicts between Protocol
-and Path Selection Properties, Path Selection Properties take precedence. For
-example, if an application indicates a preference for a specific path, but
-also a preference for a protocol not available on this path, the transport
-system will try the path first, so the Protocol Selection Property might not
-have an effect.
+use Avoided properties as a tiebreaker. 
```

Comment:
Can this conflict between Protocol and Path Selection really just be removed? We don't distinguish between these anymore in the API, but we added this because of the observation that path and protocol selection may be at odds and this might be a relevant part of the API contract, see #149. 
I'd suggest to add this again.

Reply:
Okay, will rephrase to capture the fact that we don't have two kinds of selection properties.